packages/client/components/OverflowAvatar.tsx

```diff
@@ -40,7 +41,8 @@ const OverflowCount = styled('div')<{
     : 'scale(1)',
   transition: `all 300ms ${BezierCurve.DECELERATE}`,
   userSelect: 'none',
-  width
+  width,
+  ...(onClick ? {cursor: 'pointer'} : null)
```

Comment:
+1 same as the comment above. I'd prefer to stick with the existing cursor pattern if possible

Reply:
same as https://github.com/ParabolInc/parabol/pull/6366#discussion_r877857585